back ticks in the comments in the template breaks `go build`

On running `$ go get -u github.com/gostaticanalysis/skeleton`, it fails with the following message.

```
$ go get -u github.com/gostaticanalysis/skeleton
# github.com/gostaticanalysis/skeleton
go/src/github.com/gostaticanalysis/skeleton/template.go:96:78: syntax error: unexpected ldflags, expecting comma or )
```

It is because back ticks are used inside the Go string value.
https://github.com/gostaticanalysis/skeleton/blob/master/template.go#L96

--- a/template.go
+++ b/template.go
@@ -94,7 +94,7 @@ import (
 
 // flags for Analyzer.Flag.
 // If you would like to specify flags for your plugin, you can put them via 'ldflags' as below.
-//     $ go build -buildmode=plugin -ldflags "-X 'main.flags=-opt val'" {{.ImportPath}}/plugin/{{.Pkg}
+//     $ go build -buildmode=plugin -ldflags "-X 'main.flags=-opt val'" {{.ImportPath}}/plugin/{{.Pkg}}
 var flags string
 
 // AnalyzerPlugin provides analyzers as a plugin.

--- a/template.go
+++ b/template.go
@@ -93,7 +93,7 @@ import (
 )
 
 // flags for Analyzer.Flag.
-// If you would like to specify flags for your plugin, you can put them via `ldflags` as below.
+// If you would like to specify flags for your plugin, you can put them via 'ldflags' as below.
 //     $ go build -buildmode=plugin -ldflags "-X 'main.flags=-opt val'" {{.ImportPath}}/plugin/{{.Pkg}
 var flags string
 

